src/sentry/templates/sentry/emails/error.html

```diff
@@ -32,25 +32,29 @@ <h2 style="margin-bottom: 15px">
       <div class="event-date">{{ event.datetime }} UTC</div>
     </div>
 
-    {% for label, html, _ in interfaces %}
-    <div class="interface">
-        <h3 class="title">{{ label }}</h3>
-        {{ html }}
-    </div>
-    {% endfor %}
+    {% if enhanced_privacy %}
+      <p><em>Details for this issue are unavailable due to enhanced privacy.</em></p>
```

Comment:
Something like this might be a bit clearer that the details are unavailable in the notification, but still present on the website: "Details about this issue are not shown in this notification since enhanced privacy controls are enabled. For more details about this issue, view this issue on Sentry."